scripts/issue-flux-certificates.sh

```
@@ -0,0 +1,63 @@
+
+if kubectl get secret -n cozy-system cozystack-assets-tls >/dev/null && kubectl get secret -n cozy-public cozystack-assets-tls >/dev/null; then
+  echo "Secret cozystack-assets-tls already exists in both cozy-system and cozy-public namespaces. Exiting."
+  exit 0
+fi
+
+USER_CN="cozystack-assets-reader"
+CSR_NAME="csr-${USER_CN}-$(date +%s)"
+
+# make temp directory and cleanup handler
+TMPDIR=$(mktemp -d)
+trap "rm -rf $TMPDIR" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)